src/test/java/com/epam/ta/reportportal/dao/LogRepositoryTest.java

```diff
@@ -142,6 +142,12 @@ void findIdsByItemIds() {
 		assertEquals(7, idsByTestItemIds.size());
 	}
 
+	@Test
+	void findIdsByItemIdsAndLogLevelGte() {
+		List<Long> ids = logRepository.findIdsByTestItemIdsAndLogLevelGte(Arrays.asList(1L, 2L, 3L), LogLevel.DEBUG.toInt());
```

Comment:
please, check that all logs level are really gte DEBUG

Reply:
updated